package.json

```diff
@@ -96,7 +96,7 @@
     "sass-loader": "^8.0.2",
     "semantic-release": "^18.0.0",
     "style-loader": "^1.1.3",
-    "webex": "^1.149.2",
+    "webex": "^1.161.0",
```

Comment:
Why are we moving to 1.161? Why not 2.54.0. There are no breaking changes between 1.x.x and 2.x.x.

Reply:
+1

Reply:
@arun3528, @sreenara, I was going with minor updates and testing,

Reply:
@arun3528, @sreenara, I have amended it with the latest version available, I will test the flows and update.